airflow/providers/microsoft/azure/hooks/wasb.py

```diff
@@ -311,23 +353,35 @@ def download(
         blob_client = self._get_blob_client(container_name, blob_name)
         return blob_client.download_blob(offset=offset, length=length, **kwargs)
 
-    def create_container(self, container_name: str) -> ContainerClient:
+    def create_container(self, container_name: str) -> None:
         """
         Create container object if not already existing
 
         :param container_name: The name of the container to create
         """
         container_client = self._get_container_client(container_name)
         try:
-            self.log.debug('Attempting to create container: %s', container_name)
+            self.log.info('Attempting to create container: %s', container_name)
```

Comment:
The Azure Blob SDK doesn't provide a great error message when not permitted to create a container -- something like "40X: Not permitted to perform this action." -- which isn't very helpful. I thought maybe if this statement was in the logs already users would have a better sense of what's going on rather than not knowing what action they weren't permitted to do.

Since this is configurable, I figured perhaps it's a nice "now the operator is doing this"-type message to be more informative.

Reply:
We could instead add _Attempt to create container failed_ to all the error messages instead to convey the same context, while avoiding redundant logging when container creation succeeds (which should be the case most of the time).

Reply:
Excellent point. Brilliant, I'll update this too.